src/platforms/react-native/manual-setup/codepush.mdx

```diff
@@ -46,6 +46,12 @@ codePush.getUpdateMetadata().then(update => {
 });
 ```
 
+In case you are wrapping your Root Component with `Sentry.wrap` to gain Performance features, `codePush` should be the outer most function because it needs access to the very root component in order to swap out the bundle.
```

Comment:
I'm don't understand what you mean by "the very root component". Can you explain that or try wording it another way?

```suggestion
If you're wrapping your root component with `Sentry.wrap` to access Performance features, `codePush` should be the outer most function because it needs access to the very root component in order to swap out the bundle.
```